configs/ARM/BeagleBone/Rostock-4DOF-CRAMPS/run.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
+
+import sys
+import os
+import subprocess
+import time
+from machinekit import launcher
+
+launcher.register_exit_handler()
+#launcher.set_debug_level(5)
```

Comment:
same comment about comments goes for this line, and line 17

Reply:
removed.